keyboards/varanidae/info.json

```diff
@@ -0,0 +1,156 @@
+{
+    "keyboard_name": "Varanidae",
+    "manufacturer": "Tokage IT Lab.",
+    "url": "https://github.com/TokageItLab/Varanidae",
+    "maintainer": "Silc Renew",
+    "usb": {
+        "vid": "0x5369",
+        "pid": "0x0001",
+        "device_version": "1.0.0"
+    },
+    "pin_compatible": "elite_c",
+    "development_board": "elite_c",
+    "features": {
+        "bootmagic": true,
+        "extrakey": true,
+        "mousekey": true,
+        "encoder": true
+    },
+    "encoder": {
+        "enabled": true,
+        "rotary": [
+            { "pin_a": "D0", "pin_b": "D1", "resolution": 2 }
+        ]
+    },
+    "dynamic_keymap": {
+        "layer_count": 3
+    },
+    "diode_direction": "COL2ROW",
+    "matrix_pins": {
+        "cols": ["D2", "F4", "D4", "D7", "B4", "B7", "C7", "F0", "B2", "B1", "F6"],
+        "rows": ["B0", "F5", "C6", "E6", "B5", "D5", "F1", "B6", "B3", "F7", "D3"]
+    },
+    "layouts": {
+        "LAYOUT": {
```

Comment:
Apparently your keyboard supports multiple possible layouts (there are options for split left Shift, split right Shift, split Backspace, ISO Enter).  The way to support such keyboards in QMK is:

1. Create a layout definition which covers all possible matrix positions, named `LAYOUT_all`.
   
   The current `LAYOUT` definition already does that, so the only thing needed is to rename it to `LAYOUT_all`.

2. Create some layout definitions for commonly used layouts (covering all possible permutations is not required, but at least the common ANSI and ISO layout options should be covered).

   For this keyboard probably at least definitions like `LAYOUT_ansi` and `LAYOUT_iso` should be supplied (just copy the `LAYOUT_all` definition and adjust the coordinates and sizes of affected keys, removing the keys which are not present in the specific layout):
   - `LAYOUT_ansi` — ANSI Enter, 2u Backspace, 2.25u left Shift, 2.75u right Shift.
   - `LAYOUT_iso` — ISO Enter, 2u Backspace, split (1.75u+1u) left Shift, 2.75u right Shift.

   Although given that split right Shift is advertised as an important feature of your keyboard, you may decide to supply something like `LAYOUT_ansi_split_bs_rshift` and `LAYOUT_iso_split_bs_rshift` too.

   It is not required to supply specific keymaps for all those layouts (your keyboard is wired so that the default keymap should give reasonable results for non-split Shift/Backspace and ISO Enter); you may decide to add some just to test them though.

Note that for ISO layouts the ISO Enter key should be placed in the layout at the bottom of two rows it occupies (so that it ends up on the same row as ANSI Enter would); the key ordering would be like this:
![ISO Enter key ordering](https://github.com/qmk/qmk_firmware/assets/616082/541c021a-99d7-409e-8848-a06b065b1692)


```suggestion
        "LAYOUT_all": {
```